docs: add lighthouse to package link to readme (#613)
code-pushup/cli: README.md

  - [🧩 @code-pushup/eslint-plugin](./packages/plugin-eslint#readme) - static analysis using **ESLint** rules
  - [🧩 @code-pushup/coverage-plugin](./packages/plugin-coverage#readme) - code coverage analysis
  - [🧩 @code-pushup/js-packages-plugin](./packages/plugin-js-packages#readme) - package audit and outdated dependencies

>   - [🧩 @code-pushup/lighthouse-plugin](./packages/plugin-lighthouse#readme) - standards, performance metrics, and best practices audits for the web

Reviewer: ```suggestion
  - [🧩 @code-pushup/lighthouse-plugin](./packages/plugin-lighthouse#readme) - web performance and best practices from **Lighthouse**
```
